api/lib/domain/models/User.js

```diff
@@ -74,6 +76,11 @@ class User {
     return pixAuthenticationMethod ? pixAuthenticationMethod.authenticationComplement.shouldChangePassword : null;
   }
 
+  get shouldSeeDataProtectionPolicyInformationBanner() {
```

Comment:
Remarque : on pourrait peut-être ajouté ici la condition qui dit que la bannière ne s'affiche pas pour les élèves ? Sur le front on a `const isNotAStudent = this.currentUser.user.cgu === true;`
Question : on n'a pas d'autres moyen que les cgu à false pour identifier qu'un User est de type Organisation Learner ?